src/main/java/uk/gov/hmcts/opal/dto/legacy/DefendantAccountSearchCriteria.java

```diff
@@ -16,37 +17,50 @@
 @AllArgsConstructor
 public class DefendantAccountSearchCriteria implements ToJsonString {
 
+    @JsonInclude(JsonInclude.Include.NON_NULL)
     @JsonProperty("account_number")
     private String accountNumber;
 
+    @JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
Is there an option to configure this on class level for all the fields?

Reply:
Yes - have done that in new commit